person_class.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', _parent_permission: true)
+    @id = nil
+    @name = name
+    @age = age
+  end
```

Comment:
- [ ] Please make sure to create an instance variable for parent_permission as well otherwise it will not be available in the can_use_services method.